tests/codegen/snapshots/python/union.py

```diff
@@ -8,5 +8,14 @@ class OperationNameResultUnionPerson:
 
 OperationNameResultUnion = Union[OperationNameResultUnionAnimal, OperationNameResultUnionPerson]
 
+class OperationNameResultOptionalUnionAnimal:
+    age: int
+
+class OperationNameResultOptionalUnionPerson:
+    name: str
+
+OperationNameResultOptionalUnion = Union[OperationNameResultOptionalUnionAnimal, OperationNameResultOptionalUnionPerson]
```

Comment:
Did these files get linted?

Reply:
nope, they shouldn't be :)

Reply:
Oh, codegen... 🤦

Reply:
yup, these files are snapshots, formatting them will break the tests :D

Reply:
Compare ASTs 😎